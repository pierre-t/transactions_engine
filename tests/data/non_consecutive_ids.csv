type,client,tx,amount
deposit,9988,9988,9988.77
deposit,1,1000,1000.0
deposit,2,2,2000.0
deposit,1,333,500.0
withdrawal,1,41,200.0
withdrawal,2,59090909,1000.0
dispute,1,1000,
withdrawal,1,6789,100.0
resolve,1,1000,
deposit,1,12345,3000000000.0
dispute,1,333,
chargeback,1,333,
deposit,65535,101,1000.0
withdrawal,65535,4294967295,0.111111111