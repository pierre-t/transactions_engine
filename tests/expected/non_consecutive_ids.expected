client,available,held,total,locked
1,3000000700,0,3000000700,true
2,1000,0,1000,false
9988,9988.77,0,9988.77,false
65535,999.8889,0,999.8889,false
